src/components/accordion-item/daikin-accordion-item.ts

```diff
@@ -3,64 +3,52 @@ import { LitElement, css, html, unsafeCSS, type PropertyValues } from "lit";
 import { customElement, property, state } from "lit/decorators.js";
 import { createRef, ref } from "lit/directives/ref.js";
 import tailwindStyles from "../../tailwind.css?inline";
-
-const cvaDetails = cva(
-  ["flex", "w-full", "h-max", "bg-white", "font-daikinSerif", "overflow-clip"],
-  {
-    variants: {
-      status: {
-        enabled: [],
-        disabled: ["text-[#DCDCDC]"],
-      },
-    },
-  }
-);
+import "../icon/daikin-icon";
 
 const cvaSummary = cva(
   [
     "flex",
+    "justify-between",
     "items-center",
+    "gap-2",
     "w-full",
-    "h-12",
-    "px-5",
-    "outline-none",
+    "min-h-12",
+    "py-3",
+    "pr-3",
+    "pl-4",
     "relative",
-
-    "after:block",
-    "after:w-5",
-    "after:h-5",
-    "after:m-auto",
-    "after:top-0",
-    "after:right-4",
-    "after:bottom-0",
-    "after:absolute",
-    "after:i-daikin-accordion-chevron-up",
-    "after:transition-all",
+    "focus-visible:outline",
+    "focus-visible:outline-2",
+    "focus-visible:-outline-offset-2",
+    "focus-visible:outline-daikinBlue-700",
   ],
   {
     variants: {
-      visible: {
-        open: ["after:rotate-0"],
-        close: ["after:-rotate-180"],
-      },
-      status: {
-        enabled: [
-          "hover:bg-[#DCDCDC]",
-          "hover:bg-[#DCDCDC]",
+      disabled: {
+        false: [
+          "hover:bg-daikinNeutral-100",
           "hover:cursor-pointer",
-          "focus-visible:outline",
-          "focus-visible:outline-[2px]",
-          "focus-visible:outline-[#0097E0]",
-          "focus-visible:outline-offset-[-2px]",
-
-          "after:text-[#828282]",
+          "active:bg-daikinNeutral-200",
         ],
-        disabled: ["text-[#DCDCDC]", "after:text-[#DCDCDC]"],
+        true: ["text-daikinNeutral-200", "[&>*]:text-daikinNeutral-200"],
```

Comment:
TailwindCSS has `*:text-daikinNeutral-200` notation.
BTW is it necessary? I thought it to be inherited.

```suggestion
        true: ["text-daikinNeutral-200"],
```